src/cpp/session/SessionConsoleProcessSocket.cpp

```diff
@@ -87,9 +87,35 @@ Error ConsoleProcessSocket::ensureServerRunning()
    try
    {
       pwsServer_.reset(new terminalServer());
+     
+      switch (session::options().webSocketLogLevel())
+      {
+         case 0:
+         default:
+            pwsServer_->set_access_channels(websocketpp::log::alevel::none);
+            pwsServer_->set_error_channels(websocketpp::log::elevel::none);
+            break;
+         case 1:
+            pwsServer_->set_access_channels(websocketpp::log::alevel::none);
+            pwsServer_->set_error_channels(websocketpp::log::elevel::all);
+            break;
+         case 2:
+            pwsServer_->set_access_channels(websocketpp::log::alevel::access_core);
+            pwsServer_->set_error_channels(websocketpp::log::elevel::none);
+            break;
+         case 3:
+            pwsServer_->set_access_channels(websocketpp::log::alevel::all);
+            pwsServer_->set_error_channels(websocketpp::log::elevel::all);
+            break;
+      }
 
-      pwsServer_->set_access_channels(websocketpp::log::alevel::none);
       pwsServer_->init_asio();
+      
+      // Increase websocketpp handshake timeouts to 15 seconds (default is 5). Could set to
+      // zero and eliminate altogether, but they could be helpful in reducing load under a
+      // DOS attack.
+      pwsServer_->set_open_handshake_timeout(15000);
```

Comment:
Do you think these should also be configurable? (in the same manner as `kWebSocketConnectTimeout`)